__tests__/app.test.js

```diff
@@ -0,0 +1,43 @@
+const request = require('supertest')
+const app = require('../app')
+const db = require('../db/connection.js')
+const seed = require('../db/seeds/seed')
+const testData = require('../db/data/test-data')
+
+beforeEach(() => seed(testData))
+afterAll(() => db.end())
+
+
+describe("app", () => {
+    describe("GET /api/articles", () => {
+        test("should return an array of objects with properties", () => {
+            return request(app).get('/api/articles').expect(200).then((res) => {
+                expect(res.body.articles[0]).toMatchObject({
+                    title: expect.any(String),
+                    topic: expect.any(String),
+                    author: expect.any(String),
+                    article_id: expect.any(Number),
+                    created_at: expect.any(String),
+                    votes: expect.any(Number),
+                    article_img_url: expect.any(String),
+                    comment_count: expect.any(String),
```

Comment:
It could be good to add a test to check the newly added comment_count collumn is giving us what we expect. We could for instance check a specific object like [0] and check the comment count is as expected.